org.scala-ide.sdt.debug.tests/src/org/scalaide/debug/internal/launching/LibraryJarInBootstrapTest.scala

```diff
@@ -36,6 +37,7 @@ class LibraryJarInBootstrapTest {
    * Before the fix, if order of the Scala library container and the JRE container was 'wrong', the Scala library was not added
    * to the boot classpath for test execution (nor the classpath) and the launch would fail.
    */
+  @Ignore("fails on jenkins, run it locally with maven or eclipse")
```

Comment:
Please use the same ignore message that is used in a different PR: https://github.com/scala-ide/scala-ide/pull/965/files#diff-1c306650416a4aca4ba9e9efd76227adR27 It is more detailed.